lib/download.js

```diff
@@ -28,4 +31,18 @@ function download(url, path, opts) {
   })
 }
 
+function makeAgent() {
+  let host = process.env.HEROKU_HTTP_PROXY_HOST
+
+  if (!host) {
+    return
+  }
+
+  let port = process.env.HEROKU_HTTP_PROXY_PORT || 8080
+  let auth = process.env.HEROKU_HTTP_PROXY_AUTH
```

Comment:
Should there be some default here? Not sure what values are allowed in the options. Also, if we called this variable `proxyAuth`, we could use property value shorthand for everything on line 43. I suppose we would, however, sacrifice the nice alignment with line 41, lol. Guess I'll leave that call to you.

Reply:
> Should there be some default here? Not sure what values are allowed in the options. 

That’s a good question. I copied this code mostly 1:1 [from heroku-client](https://github.com/heroku/node-heroku-client/blob/0e3efffef006ea17003640de0f8d6e5f71f791a8/lib/request.js#L58-L60).

> we could use property value shorthand for everything on line 43. I suppose we would, however, sacrifice the nice alignment with line 41, lol. Guess I'll leave that call to you.

This caused me no small amount of consternation when I wrote it 😅

Reply:
I think I’ll ship this as-is rather than refining further. We should rewrite this as an OCLIF plugin at some point anyways.